components/feature/contextmenu/src/main/res/values/strings.xml

```diff
@@ -25,4 +25,5 @@
     <string name="mozac_feature_contextmenu_snackbar_link_copied">Link copied to clipboard</string>
     <!-- Action shown in a "snacbkar" after opening a new/private tab. Clicking this action will switch to the newly opened tab. -->
     <string name="mozac_feature_contextmenu_snackbar_action_switch">Switch</string>
+    <string name="mozac_feature_contextmenu_open_link_in_external_app">Open link in external app</string>
```

Comment:
Can you please add screenshot to review so we can have UX review the string? Thanks,

Reply:
Added :)